src/PharoThings-Devices-HD44780.package/PotHD44780Controller.class/instance/showMessage..st

```diff
@@ -0,0 +1,8 @@
+writing
+showMessage: aString
+	| charHex |
+	1 to: aString size do: [ :i | 
+		charHex := (aString at: i) asciiValue.
+		(aString at: i) = $\
+			ifTrue: [ self setCursorAtRow: 2 ]
+			ifFalse: [ self writeText: charHex ] ]
```

Comment:
And I think #showMethod: does not requires this #asciiValue logic.
So it would be more clear to rewrite method as:
```Smalltalk
showMessage: aString
	aString do: [:char |
              char = Character cr
			ifTrue: [ self setCursorAtRow: 2 ]
			ifFalse: [ self writeCharacter: char ] ]
```
And method should count rows to use correct value instead of hardcoded 2 (setCursorAtRow: 2)